reference/gdscript.rst

```diff
@@ -27,32 +28,32 @@ enormous amount of code. After some experiments with
 
 The last third party scripting language that was used for shipped games
 was `Squirrel <http://squirrel-lang.org>`__, but it was dropped as well.
-At that point, it became evident that Godot would work more optimally by
-using a built-in scripting language, as the following barriers were met:
+At that point, it became evident that a custom scripting language could 
+more optimally make use of Godots particular architecture:
```

Comment:
I'd also use "would" instead of "could" following "it became evident", but that's detail.